dataeng/jobs/analytics/SnowflakeMicrobachelorsITK.groovy

```diff
@@ -29,6 +29,10 @@ class SnowflakeMicrobachelorsITK {
                 env('PASSPHRASE_PATH', allVars.get('PASSPHRASE_PATH'))
                 env('USER', allVars.get('USER'))
                 env('ACCOUNT', allVars.get('ACCOUNT'))
+                env('SFTP_HOSTNAME', allVars.get('SFTP_HOSTNAME'))
+                env('SFTP_USER', allVars.get('SFTP_USER'))
+                env('SFTP_PASSWORD', allVars.get('SFTP_PASSWORD'))
+                env('SFTP_PATH', allVars.get('SFTP_PATH'))
```

Comment:
I think we may need two environment variables for this? Something along the lines of SFTP_PATH_COURSE and SFTP_PATH_STUDENT (SFTP_PATH_RETENTION?). Each report has it's own dedicated directory on ITK's file server.

Reply:
Fixed!